Configuration/Config.cs

```diff
@@ -181,6 +181,44 @@ public static void Set(string key, string value)
             environment[key] = value;
         }
 
+        /// <summary>
+        /// Sets new configuration value at the specified path. If some of the properties
+        /// that constitute a path do not exist method will create them.
+        /// </summary>
+        /// <param name="path">Json value path. A path could be multi-levelled (separated by commas).</param>
+        /// <param name="value">Value to set.</param>
+        public static void SetValue(string path, string value)
```

Comment:
How will this behave regarding multiple loan environments? I suppose they would be handled transparently by simply providing a path that is environment aware.

Reply:
It does not affect environments - yes - would just help to store the value at precise path provided. If path does not exist this will create it. I thought this could be helpful to merge some logically related variables within a single Json object. For example.. to set this 
  "parameters": {
    "intrinio-username": "",
    "intrinio-password": "",
    "ema-fast": 10,
    "ema-slow": 20
  },

I could call 

Config.SetValue("parameters.intrinio-username", "");
Config.SetValue("parameters.ema-slow", "20");
etc. 

Current Set will throw an exception if I try set any of above. This line will throw an error:
environment[key] = value;

And this is likely because JsonNet indexer would not let to set value with complex path but works fine with simple. This would be ok:
Config.Set("alpaca-trading-mode", "paper")